vendor.conf

```diff
@@ -10,7 +10,7 @@ github.com/BurntSushi/toml                          3012a1dbe2e4bd1391d42b32f057
 github.com/cespare/xxhash/v2                        d7df74196a9e781ede915320c11c378c1b2f3a1f # v2.1.1
 github.com/containerd/cgroups                       7347743e5d1e8500d9f27c8e748e689ed991d92b
 github.com/containerd/console                       8375c3424e4d7b114e8a90a4a40c8e1b40d1d4e6
-github.com/containerd/containerd                    01310155947cb6eec37dcae29742a165e56acb4a
+github.com/containerd/containerd                    ensure_remove_all https://github.com/thaJeztah/containerd.git
```

Comment:
temporarily using my fork until https://github.com/containerd/containerd/pull/4096 is merged